tests/meta/clean/test_clean_flow_cells_api.py

```diff
@@ -0,0 +1,246 @@
+    assert not flow_cell_clean_api_can_be_removed.is_flow_cell_backed_up()
+
+
+def test_get_sequencing_metrics_for_flow_cell_from_statusdb(
+    flow_cell_clean_api_can_be_removed: CleanFlowCellAPI,
+):
+    """Test to get a sequencing metrics for a flow cell from statusdb."""
+
+    # GIVEN a clean flow cell api with a store that contains a flow cell with SampleLaneSequencingMetrics entries
+
+    # WHEN retrieving the flow cell from the store
+    metrics: List[
+        SampleLaneSequencingMetrics
+    ] = flow_cell_clean_api_can_be_removed.get_sequencing_metrics_for_flow_cell()
+
+    # THEN a SampleLaneSequencingMetrics entry is retrieved
+    assert isinstance(metrics[0], SampleLaneSequencingMetrics)
+
+
+def test_has_sequencing_metrics_in_statusdb(flow_cell_clean_api_can_be_removed: CleanFlowCellAPI):
+    """Test to check that a flow cell has SampleLaneSequencingMetrics entries in statusdb."""
+
+    # GIVEN a clean flow cell api with a store that contains a flow cell with SampleLaneSequencingMetrics entries
+
+    # THEN checking whether a flow cell has SampleLaneSequencingMetrics entries returns True
+    assert flow_cell_clean_api_can_be_removed.has_sequencing_metrics_in_statusdb()
+
+
+def test_is_directory_older_than_21_days_pass(
+    flow_cell_clean_api_can_be_removed: CleanFlowCellAPI,
+):
+    """Test to check whether a directory is older than 21 days."""
+
+    # GIVEN a clean flow cell api with a flow cell that can be deleted
+
+    # THEN checking whether a given flow cell directory is older than 21 days is TRUE
+    with mock.patch(
+        "time.time",
+        return_value=time.time() + TWENTY_ONE_DAYS_IN_SECONDS,
+    ):
+        assert flow_cell_clean_api_can_be_removed.is_directory_older_than_21_days()
+
+
+def test_is_directory_older_than_21_days_fail(flow_cell_clean_api_can_be_removed: CleanFlowCellAPI):
+    """Test to check whether a directory is older than 21 days."""
+
+    # GIVEN a clean flow cell api with a current time that is set to now.
+
+    # THEN checking whether a given flow cell directory is older than 21 days is FALSE
+    assert not flow_cell_clean_api_can_be_removed.is_directory_older_than_21_days()
+
+
+def test_has_sample_sheet_in_housekeeper(flow_cell_clean_api_can_be_removed: CleanFlowCellAPI):
+    """Test to check whether a flow cell has a sample sheet in housekeeper."""
+
+    # GIVEN a clean flow cell api with a flow cell that has a sample sheet in housekeeper
+
+    # THEN checking whether the flow cell has a sample sheet in housekeeper returns TRUE
+    assert flow_cell_clean_api_can_be_removed.has_sample_sheet_in_housekeeper()
+
+
+def test_get_files_for_flow_cell_bundle(flow_cell_clean_api_can_be_removed: CleanFlowCellAPI):
+    """Test to get files for a flow cell bundle from housekeeper."""
+
+    # GIVEN a clean flow cell api with a flow cell that has files in housekeeper
+
+    # WHEN getting fastq and spring files that are tagged with the flow cell
```

Comment:
It would be cool to parametrise this test, but feel free to disregard

Reply:
fixed it